google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/CompositeTracer.java

```diff
@@ -23,8 +24,10 @@
 import org.threeten.bp.Duration;
 
 /** Combines multiple {@link ApiTracer}s into a single {@link ApiTracer}. */
-class CompositeTracer extends BaseApiTracer {
+@InternalApi("For internal use only")
+public class CompositeTracer extends BaseApiTracer {
```

Comment:
We should consider renaming this. Maybe BigtableTracer? And the doc should be updated to reflect its responsibilities, something like:
A Bigtable specific tracer that will be used to plumb additional context through the call chains as well as delegating to user defined tracers. This will ensure that operation lifecycle events are plumbed through while maintaining user configured functionationality